components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -452,6 +454,61 @@ struct AssetPrice {
   string asset_timeframe_change;
 };
 
+struct LogoImages {
+  string dark_url;
+  string dark_short_url;
+  string light_url;
+  string light_short_url;
+};
+
+struct ServiceProvider {
+  string name;
+  string service_provider;
+  string status;
+  string web_site_url;
+  LogoImages logo_images;
+};
+
+struct CryptoQuote {
+  string transaction_type;
+  double exchange_rate;
```

Comment:
Refer to my comment on conversion callbacks. If possible let's use `string` type for all numeric types, which makes it safer to use in the frontend.